components/Heading/Heading.css

```diff
@@ -19,25 +33,79 @@
 }
 
 .secondary {
-  color: var(--secondary);
+  background-color: var(--secondary);
```

Comment:
Don't define `background-color` on any of these, and you won't need `.transparentWhite` defined.

Then you can ditch the iconFill classes... In fact, I think I mislead you here. We should probably just drop theme-handling styles in this component. We should define `fill` instead the theme styles of the Container component.

Reply:
I'm not sure where you are going with this. Where do you want to handle passing in themes then? I needed transparentWhite for the Donate section with the flags in the background.

Reply:
The only reason `Heading` takes `theme` as a prop is because it's passed through by `Container`.

It only needs access to theme to define the header's text color, right?